README.md

```diff
@@ -472,6 +472,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Hygraph](https://hygraph.com/) - Offers free tier for small projects. GraphQL first API. Move away from legacy solutions to the GraphQL native Headless CMS - and deliver omnichannel content API first.
   * [Squidex](https://squidex.io/) - Offers free tier for small projects. API / GraphQL first. Open source, and based on event sourcing (versing every changes automatically).
   * [InstaWP](https://instawp.com/) - Launch a WordPress site in a few seconds. A free tier with: 5 Active Sites, 500 MB Space, 48 hrs Site Expiry.
+  * [Umbraco CMS](https://umbraco.com/) - Fully extendable CMS based on ASP.NET Core.
```

Comment:
Please describe the free features of the SaaS offering.

Reply:
The product itself is free, if you host it yourself.
If you choose the SaaS product there is no free plan.

The product is based on ASP.NET Core and it is open-source.

Reply:
Then just delete the pull request.

I didn't get that the 2 requirements was linked.